doc/ref/configuration/minion.rst

```diff
@@ -81,9 +81,10 @@ The option can can also be set to a list of masters, enabling
 ``ipv6``
 --------
 
-Default: ``False``
+Default: ``None``
```

Comment:
You both make very strong points. I appreciate the explanation and you taking the time to walk me through your reasoning. We'll get this in.